rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -213,7 +231,7 @@ def __init__(self,
                  device_id=0,):
         pipe = pipelines
         super(ROCALClassificationIterator, self).__init__(pipe, tensor_layout=pipe._tensor_layout, tensor_dtype=pipe._tensor_dtype,
-                                                          multiplier=pipe._multiplier, offset=pipe._offset, display=display, device=device, device_id=device_id)
+                                                          multiplier=pipe._multiplier, offset=pipe._offset, display=display, device=device, device_id=device_id, size = size, auto_reset = auto_reset)
```

Comment:
There is space added before and after = operator for some instances and for few instances it is not added

Reply:
Removed the spaces for newly added variables